2-ui/1-document/07-modifying-document/article.md

```diff
@@ -257,30 +257,30 @@ For instance:
 </script>
 ```
 
-...Would lead to:
+...Suna yonlendirir:
```

Comment:
Şuna*

Reply:
:) Klavyemde Türkçe karakterler yok. Bu repo için görsel klavye kullanacağım